docs/tutorials/multi-product-formula.ipynb

```diff
@@ -0,0 +1,2620 @@
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 17,
+   "id": "87323ed2",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from qiskit_addon_utils.slicing import slice_by_depth\n",
+    "from qiskit_addon_mpf.backends.tenpy_layers import LayerModel\n",
+    "from qiskit_addon_mpf.backends.tenpy_layers import LayerwiseEvolver\n",
+    "from functools import partial\n",
+    "\n",
+    "# Create approximate time-evolution circuits\n",
+    "single_2nd_order_circ = generate_time_evolution_circuit(\n",
+    "    hamiltonian, time=1.0, synthesis=SuzukiTrotter(reps=1, order=order)\n",
+    ")\n",
+    "single_2nd_order_circ = pm.run(single_2nd_order_circ)  # collect XX and YY\n",
+    "\n",
+    "# Find layers in the circuit\n",
+    "layers = slice_by_depth(single_2nd_order_circ, max_slice_depth=1)\n",
+    "\n",
+    "# Create tensor network models\n",
+    "models = [\n",
+    "    LayerModel.from_quantum_circuit(layer, conserve=\"Sz\") for layer in layers\n",
+    "]\n",
+    "\n",
+    "# Create the time-evolution object\n",
+    "approx_factory = partial(\n",
+    "    LayerwiseEvolver,\n",
+    "    layers=models,\n",
+    "    options={\n",
+    "        \"preserve_norm\": False,\n",
+    "        \"trunc_params\": {\n",
+    "            \"chi_max\": 64,\n",
+    "            \"svd_min\": 1e-8,\n",
+    "            \"trunc_cut\": None,\n",
+    "        },\n",
+    "        \"max_delta_t\": 2,\n",
+    "    },\n",
+    ")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "fd39f15a",
+   "metadata": {},
+   "source": [
+    "<div class=\"alert alert-block alert-danger\">\n",
+    "The options of LayerwiseEvolver which determine the details of the tensor network simulation must be chosen carefully to avoid setting up an ill-defined optimization problem\n",
```

Comment:
```suggestion
    "The options of LayerwiseEvolver that determine the details of the tensor network simulation must be chosen carefully to avoid setting up an ill-defined optimization problem.\n",
```